apps/web/src/features/layout/components/ThemeEnvironment/oceanEnvironment.module.css

```
@@ -0,0 +1,238 @@
+  max-width: 320px;
+  box-shadow:
+    0 8px 32px rgba(0, 240, 255, 0.25),
+    inset 0 1px 0 rgba(255, 255, 255, 0.1);
+  color: #f0f9ff;
+  font-family: "Plus Jakarta Sans", system-ui, sans-serif;
+  z-index: 50;
+  transition: opacity 0.2s ease, transform 0.2s ease;
+  animation: tooltipFadeIn 0.25s cubic-bezier(0.16, 1, 0.3, 1) forwards;
+}
+
+@keyframes tooltipFadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "tooltipFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)